packages/client/src/Components/Mapping/helpers/planningRouteFor.js

```diff
@@ -1,13 +1,20 @@
 import L from 'leaflet'
 
 // the images we use to annotate planning legs
-import turnLeft from '../images/turn-left.png'
-import turnRight from '../images/turn-right.png'
-import turnEnd from '../images/turn-end.png'
+import turn0deg from '../images/turn-vert.png'
```

Comment:
Please name the icons files using the numbering scheme you're using for the variables